app/src/main/res/values/strings.xml

```diff
@@ -469,7 +469,7 @@
     <string name="visits_overview_client_name_column_label">Child Name</string>
     <string name="visits_overview_details_visit_date_label">Visit Date:</string>
     <string name="visits_overview_details_scheduled_visit_date_label">Scheduled Visit Date:</string>
-    <string name="visits_overview_details_historical_visit_date_label">Historical Visit Date:</string>
+    <string name="visits_overview_details_historical_visit_date_label">Today\'s Visit Date:</string>
```

Comment:
this label change does not make sense. This label is displayed in historical visits so it will be shown only for past visits so label "Today's Visit date" does not make sense

Reply:
This was a requirement from Eva and Francis, they agreed with that kind of Naming.